src/com/crawljax/core/Crawler.java

```diff
@@ -188,8 +188,7 @@ private void tryToFireEvent(StateVertex targetState, StateVertex curState, Event
 			plugins.runOnRevisitStatePlugins(context, curState);
 
 		} else {
-			throw new StateUnreachableException(targetState, "couldn't fire eventable "
-			        + clickable);
+			throw new StateUnreachableException(targetState, "couldn't fire eventable " + clickable.toString());
```

Comment:
The realò trouble is that Netbeans doesn't compile without this... because it's recognized as a conflict and is not considered as a string... this is the only solution that allow Netbeans to manage the software unfortunately